zyte_common_items/processors.py

```diff
@@ -31,6 +31,26 @@
 )
 
 
+def _to_int(value: Any) -> Any:
+    if isinstance(value, Integral):
+        return int(value)
+    elif isinstance(value, str):
+        if "," in value:
+            value = value.replace(",", "")
+        return int(value)
+    return value
+
+
+def _to_float(value: Any) -> Any:
+    if isinstance(value, Real):
+        return float(value)
+    elif isinstance(value, str):
+        if "," in value:
+            value = value.replace(",", ".")
```

Comment:
nitpick: why is `if` necesary? Is it an optimization, and if so - does it actually help (it can go both ways - it's 2 passes over string instead of one)?